glibc.spec

```diff
@@ -13,15 +25,15 @@ Source: git+https://github.com/%{github_user}/glibc.git?obj=%{branch}/%{tag}&exp
 
 %build
 
-rm -rf ../glibc-build
-mkdir ../glibc-build
-cd ../glibc-build
-../glibc-%{realversion}/configure \
+rm -rf ../%{n}-build
+mkdir ../%{n}-build
+cd ../%{n}-build
+../%{n}-%{realversion}/configure \
   CC=gcc \
   CXX=g++ \
   CFLAGS='-mtune=generic -fasynchronous-unwind-tables -DNDEBUG -g -O3' \
   --prefix=/usr \
-  --enable-add-ons=nptl,rtkaio,c_stubs,libidn \
```

Comment:
These flags were picked by rebuilding glibc with _rpmbuild_ or _mock_ to match exactly what was used in distro. Have you cross-checked this by building glibc on CC7? This is to make sure that we are not breaking system is some way.